### ISSUE TITLE
Catboost's support for loss_function:mse

### ISSUE THREAD

**Issue by iFe1er** [catboost/catboost#140]
According to the document:
https://tech.yandex.com/catboost/doc/dg/concepts/loss-functions-docpage/#loss-functions  

Catboost doesn't support MSE as loss function, whereas LightGBM support regression_l2 and XGBoost use a overall regression objective: reg:linear

I wonder why Catboost can not support a "regression_l2" ike Lightgbm or use MSE as a loss function.

Moreover, I try catboost on kaggle open source data: https://www.kaggle.com/c/house-prices-advanced-regression-techniques/data    

The data required a RMSE as objective, and i tried two different parameter set for this problem:

First:
        params = {}
        params['loss_function'] = 'RMSE'
        params['eval_metric'] = 'RMSE'
        params['iterations'] = 300
        params['depth'] = 4
        params['learning_rate'] = 0.2  
        params['od_type'] = 'Iter'
        params['od_wait'] = 10

Second:
        params = {}
        params['loss_function'] = 'MAE'
        params['eval_metric'] = 'RMSE'
        params['iterations'] = 300
        params['depth'] = 4
        params['learning_rate'] = 0.2
        params['od_type'] = 'Iter'
        params['od_wait'] = 10

and then use catboost.cv for testing.

I thought RMSE is more suitable for a problem which objective is RMSE.  

However ,the MAE loss function do MUCH BETTER than RMSE one. (0.15 VS 0.20), whereas LightGBM can easily achieve ~0.12. What happened?

**Comment by annaveronika** [comment]
Actually we optimize MSE internally. About your results: maybe you need more iterations, check metric value on test and look if metric has stopped decreasing. Anyway the results will depend on the dataset.

**Comment by annaveronika** [comment]
I also would advise to read about parameter tunning for catboost here: https://tech.yandex.com/catboost/doc/dg/concepts/parameter-tuning-docpage/

**Comment by ZacheryAU** [comment]
Hi Anna, 
I know that optimize MSE is same as RMSE, but we just want to compare lightgbm with catboost simultaneously during training. However, lightgbm only shows MSE while catboost only shows RMSE. Is it a kidding?
